ntbs-service/Migrations/20191218162926_UpdateDiseaseSites.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using ntbs_service.Models.ReferenceEntities;
+
+namespace ntbs_service.Migrations
+{
+    public partial class UpdateDiseaseSites : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // Remove any 'Unknown' disease sites as we are removing this as an option
+            migrationBuilder.Sql("DELETE FROM [NotificationSite] WHERE SiteId = 18");
```

Comment:
Didn't see much merit in shifting this out to another migration, so dumped here. If there's a preference for it to be outside I can get that done.

Reply:
nah, here seems just right